src/backend/distributed/commands/view.c

```diff
@@ -541,14 +554,16 @@ PreprocessRenameViewStmt(Node *node, const char *queryString,
 	/* deparse sql*/
 	const char *renameStmtSql = DeparseTreeNode(node);
 
-	EnsureSequentialMode(OBJECT_VIEW);
-
-	/* to prevent recursion with mx we disable ddl propagation */
-	List *commands = list_make3(DISABLE_DDL_PROPAGATION,
-								(void *) renameStmtSql,
-								ENABLE_DDL_PROPAGATION);
+	/*
+	 * To not enforce sequential connection, we are using metadata connection. For the
```

Comment:
very minor: I guess `To avoid sequential mode` is probably a better fit here. (same for above)